TicTacToeUserInterface/UserInterface.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using TicTacToeApp;
+
+namespace TicTacToeUserInterface
+{
+    public class UserInterface
+    {
+
+        public TicTacToe newGame;
+
+
+        static void Main(string[] args)

```

Comment:
You have multiple `Main` methods. This one here and the one in TicTacToeApp. The `Main` method is the entry point into your app. That means when you run your executable, it will be the first method to "kick things off". You only need one for your app so you can decide where you want it to be. Without going into specifics it would look something like this:

```
A.csproj -- this is a command line application (i.e, dotnet new console ...)
B.csproj -- this is a class library (i.e, dotnet new classlib) it's not meant to be executable, it contains classes that can be shared and re-used.
C.csproj -- this is maybe another library, your app may have 1 or more libs depending on how you organize your code (i.e, dotnet new classlib)
```
`A.csproj` would depend on libraries `B` and `C` and it would be the only place where `Main` is defined.

Reply:
Okay, this is making more sense now. `B.csproj` would have the reusable business logic etc